### PR TITLE
calendar

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
```
Comment (ZUBOV-ILLIA) [inline src/styles/main.scss]: Use classes instead of tags for styling.
`body` dont have any `padding`.

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,76 @@
...
+
+.calendar {
+  display: flex;
+  padding: 20px;
+  justify-content: start;
+  flex-wrap: wrap;
+  width: (($size + 1) * 7);
```
Comment (ZUBOV-ILLIA) [inline src/styles/main.scss]: You can't set `width` to BEM block.

**Review by ZUBOV-ILLIA — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 1 above)
Comment (ISRLv2021) [reply src/styles/main.scss]: Thanks, I fixed

File: src/styles/main.scss (hunk 2 above)
Comment (ISRLv2021) [reply src/styles/main.scss]: changed

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+  "sun" (($size + 1) * 6);
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
 body {
   margin: 0;
+  padding: 0;
```
Comment (denys-cheporniuk) [inline src/styles/main.scss]: body doesn't have default padding, remove it

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+  box-sizing: border-box;
```
Comment (denys-cheporniuk) [inline src/styles/main.scss]: add `box-sizing` for  selector `*`

File: src/styles/main.scss (hunk 2 above)
Comment (denys-cheporniuk) [inline src/styles/main.scss]:
```suggestion
  width: ($size + 1) * 7;
```

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**